src/editor.js

```diff
@@ -23,7 +23,12 @@ class MonacoEditor extends React.Component {
       // Consider the situation of rendering 1+ times before the editor mounted
       if (this.editor) {
         this.__prevent_trigger_change_event = true;
-        this.editor.setValue(this.__current_value);
+        this.editor.pushUndoStop();
+        this.editor.executeEdits('', [{
+          range: this.editor.getModel().getFullModelRange(),
+          text: this.__current_value
```

Comment:
Should we set https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.iidentifiedsingleeditoperation.html#forcemovemarkers here? Can you explain what it does?

Reply:
It relates to markers and undo operations. I don't understand it much, but we probably don't need it.

Go to https://microsoft.github.io/monaco-editor/playground.html and paste the following

```
var ed = monaco.editor.create(document.getElementById("container"), {
	value: "abcdefg",
	language: "javascript"
});

monaco.editor.setModelMarkers(ed.getModel(), 'test', [{
    startLineNumber: 1,
    startColumn: 5,
    endLineNumber: 1,
    endColumn: 7,
    message: "a message",
    severity: 4
}])

ed.executeEdits('', [{
    range: new monaco.Range(1, 5, 1, 7), 
    text: "111111", 
}]);
```

Go to the code editor and perform an undo operation. We can see that the marker stays the same at the column 5-7.

Now, try doing the same with  `forceMoveMarkers: true` added. The marker somehow is moved to the end of the line instead.

Reading the documentation, I'm still confused for what it means actually. In any case, the above experiment shows that it probably won't do what we want, and probably won't matter much.